classes/book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Book < Item
+  attr_reader :publisher, :cover_state
+
+  def initialize(publish_date, publisher, cover_state)
+    super(publish_date)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  private
+
+  def can_be_archived?
+    super || cover_state == 'bad'
+  end
```

Comment:
- [ ] please kindly call downcase method on @cover_state

- [ ] please kindly add a method for appending @labels into label array